user_guides/jaxws/src/main/jbake/content/config.inc

```diff
@@ -273,15 +320,15 @@ com.sun.ts.lib.implementation.sun.common.SunRIURL
 ----
 --
 +
-  i.  Set the `user` and `password` properties to the user name and
+  h.  Set the `user` and `password` properties to the user name and
   password used for the basic authentication tests. +
   The default setting for both is `j2ee`.
-  j. Set the `authuser` and `authpassword` properties to the user name
+  i. Set the `authuser` and `authpassword` properties to the user name
   and password used for the basic authentication tests. +
   The default setting for both is `javajoe`.
-  k. Set the `http.server.supports.endpoint.publish` property based on
+  j. Set the `http.server.supports.endpoint.publish` property based on
   whether Endpoint Publish APIs are supported on the container.
-  l. If using Java SE 8 or above, verify that the property
+  k. If using Java SE 8 or above, verify that the property
```

Comment:
consider changing Java SE version ID to {SEversion}

Reply:
I have to remove this paragraph. Endorsed dir makes no sense with all packages renamed to jakarta.*

Reply:
But also did search/replace for Java SE 8 in the rest of this file.